src/mom5/ocean_core/ocean_types.F90

```diff
@@ -1097,6 +1098,7 @@ module ocean_types_mod
      real, _ALLOCATABLE, dimension(:,:,:)     :: bmf             _NULL ! momentum flux per mass into ocean bottom  (N/m^2)
      real, _ALLOCATABLE, dimension(:,:)       :: gamma           _NULL ! dimensionful bottom drag coefficient (kg/(m^2 sec))
      real, _ALLOCATABLE, dimension(:,:)       :: langmuirfactor  _NULL ! Langmuir turbulence enhancement factor
+     real, _ALLOCATABLE, dimension(:,:)       :: u10             _NULL ! x-dir surface stokes drift
```

Comment:
I believe the comment should read "10m wind speed (m/s)".

Reply:
Fixed, thanks for the Eagle Eyes Steve

https://github.com/mom-ocean/MOM5/pull/229/commits/6c27f205c77360b09cb2868eee9ad2be421358c0